package.json

```diff
@@ -90,10 +90,11 @@
     "winston": "3.3.3"
   },
   "dependencies": {
+    "@grpc/grpc-js": "^1.3.7",
```

Comment:
Can we be sure future versions of `exporter-collector-grpc` will have a compatible version of `@grpc/grpc-js`? Is the dependency explicitly needed if the exporter depends on one?

Reply:
The version of this should be updated with the exporter - this range is what was specified on the exporter.

The dep isn't explicitly needed, added it when lint pointed that out. Although, the only real reason to actually do that in our case is to avoid a situation where the exporter removes the dep which breaks us. Even if we do add the dep, it causes undefined behaviour because the exporter will not be compatible anymore.

Removing the dep and adding the lint exception makes more sense to me now - WDYT?

Reply:
See https://github.com/signalfx/splunk-otel-js/pull/319/commits/bfb7d93eb128628ed261e1b5f17a8bf768a09214.